alphagov/paas-cf
# [#120938265] Change GPG key ID for Hector Rivas Gandara

# What 

The gpg key from Hector Rivas is in risk of being compromised due lost
of a device.

The device was encrypted and the GPG key was protected by passphrase,
but we decided to rotate the related credentials anyway.

The new key has been already uploaded to the required key servers.

# How to test?

Simply check that you can get the key:

```
gpg --keyserver hkp://keyserver.ubuntu.com --search-keys 0x1447F48C
``` 

http://keyserver.ubuntu.com:11371/pks/lookup?search=0x1447F48C&op=vindex
# Who? 

Anyone but me
